crates/transaction-pool/src/pool/txpool.rs

```diff
@@ -17,7 +17,7 @@ use crate::{
     ValidPoolTransaction, U256,
```

Comment:
for `transaction-pool` we want the native `u128` type and not the `U128`.

Reply:
updated